uberfire-api/src/main/java/org/uberfire/workbench/annotations/VisibleAsset.java

```diff
@@ -0,0 +1,16 @@
+package org.uberfire.workbench.annotations;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import javax.inject.Qualifier;
+
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+@Qualifier
+@Retention(RUNTIME)
+@Target({TYPE, PARAMETER})
+public @interface VisibleAsset {
```

Comment:
Could you please add `@Documented` and javadoc to this qualifier? Things worth mentioning in the javadoc:

- What is this qualifier used for? (Don't have to mention the library, but maybe more generally that this means that this type of resource is a file that should be visible to end users.)
- Why you should still add `@Default` when using this qualifier.

Reply:
@mbarkley thanks for the comments, I will fix them right now